index.js

```diff
@@ -1,4 +1,13 @@
-let i = 0;
-this is actually not nice.
-this is actually not nice either
-this is actually nicer. I think I found the solution.
\ No newline at end of file
+// Setup
+var outerWear = "T-Shirt";
+
+function myOutfit() {
+  // Only change code below this line
+
+
+
+  // Only change code above this line
+  return outerWear;
+}
```

Comment:
you should write function as a variable. you must use this 
```suggestion
function myOutfit() {
  // Only change code below this line
  this is how it is done;



  // Only change code above this line
  return outerWear;
}
```